DOCU-2369: Removes prompts from instruction commands (#4074)
Kong/docs.konghq.com: src/kubernetes-ingress-controller/guides/getting-started.md

If everything is setup correctly, making a request to Kong should return back

> a HTTP 404 Not Found.

Reviewer: ```suggestion
a HTTP `404 Not Found` status code.
```

HTTP requests with /bar -> Kong enforces rate-limit +   -> echo-server
   on example.com          injects my-request-id header
```

## Next steps

> * To learn how to secure proxied routes, check out the [ACL and JWT Plugins Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/configure-acl-plugin/)

Reviewer: ```suggestion
* To learn how to secure proxied routes, see the [ACL and JWT Plugins Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/configure-acl-plugin/).
```

> * The [External Services Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-external-service/) explains how to proxy services outside of your Kubernetes cluster

Reviewer: ```suggestion
* The [External Services Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-external-service/) explains how to proxy services outside of your Kubernetes cluster.
```
